### PR TITLE
Add aspects to curriculum/javascript/core/numbers/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/base-conversion.md?ref=651c1135ca45080abd4a0b75bf2ab90e417b761f

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

File: javascript/core/numbers/js-practice-math-max.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ standards:
 
   javascript.standard-library.1: 1000
   javascript.write-expressions.0: 1000
+  
+aspects:
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-math-max.md]: add `introduction`

File: javascript/core/numbers/js-practice-math-pow.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/5733f948d780e27df6000e33
 linkType: codewars
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-math-pow.md]: add `introduction`

File: javascript/core/numbers/js-practice-math-log.md [hunk 3]
```diff
@@ -15,7 +15,10 @@ standards:
 
   javascript.standard-library.1: 1000
   javascript.write-expressions.0: 1000
-
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-math-log.md]: add `introduction`

File: javascript/core/numbers/infinity-game.md [hunk 4]
```diff
@@ -14,6 +14,10 @@ standards:
   javascript.data-types-structures.0: 1000
   javascript.standard-library.2: 1000
   javascript.evaluate-expressions.1: 1000
+  
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/infinity-game.md]: add `introduction`

File: javascript/core/numbers/js-practice-math-random.md [hunk 5]
```diff
@@ -15,7 +15,10 @@ standards:
 
   javascript.standard-library.1: 1000
   javascript.write-expressions.0: 1000
-
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-math-random.md]: `introduction`

File: javascript/core/numbers/js-practice-number-methods-round.md [hunk 6]
```diff
@@ -12,8 +12,9 @@ tags:
 
 
 type: exercise
-
-
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-number-methods-round.md]: `introduction`

File: javascript/core/numbers/js-practice-number-methods-tostring.md [hunk 7]
```diff
@@ -15,7 +15,9 @@ tags:
 
 type: exercise
 
-
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-number-methods-tostring.md]: `introduction`

File: javascript/core/numbers/js-practice-number-tostring.md [hunk 8]
```diff
@@ -18,7 +18,10 @@ standards:
   javascript.standard-library.2: 1000
   javascript.write-expressions.0: 1000
   javascript.evaluate-expressions.0: 1000
-
+  
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/numbers/js-practice-number-tostring.md]: `introduction`

File: javascript/core/numbers/nan-is-a-number.md [hunk 9]
```diff
@@ -10,6 +10,12 @@ type: normal
 
 category: caveats
 
+aspects:
+  - introduction
+  - new
```
Comment (ryanwarsaw) [inline javascript/core/numbers/nan-is-a-number.md]: remove `new`

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Comment by enki-hq** [comment]
Failed parsing files: 
  - javascript/core/numbers/js-practice-number-tostring.md: 
 > duplicated mapping key at line 29, column -26:
    aspects:
    ^

**Pull request merged**